src/main/telemetry/frsky_hub.c

```diff
@@ -542,16 +542,16 @@ void processFrSkyHubTelemetry(timeUs_t currentTimeUs)
 
         // Sent every 500ms
         if ((cycleNum % 4) == 0) {
-            int16_t altitude = ABS(getEstimatedAltitude());
+            int32_t altitude_cm = getEstimatedAltitude_cm();
```

Comment:
Are we sure this can handle negative altitudes? The `ABS` in the existing code suggests otherwise.

Reply:
Looking at this
https://www.frsky-rc.com/wp-content/uploads/2017/07/Manual/protocol_sensor_hub.pdf Page 3
for me GPS- and variometer(baro)-altitude are defined identically: signed
BTW I will add "... | sensors(SENSOR_GPS)" to the conditional check. Otherwise altitude won't be capture with GPS only.
A general question: sensors(SENSOR_GPS) does not seem to detect GPS at early boot time. There were some heuristic fixes in the past using features(GPS) instead. Should this generally be changed or is sensors(SENSOR_GPS) sufficient for every "slow" post-boot-code?

Reply:
Unlike other sensors, GPS detection is asynchronous on boot, and the boot sequence will continue before the first serial packet from the GPS allows the detection to complete. Therefore, one-off checks at init time should use `features(FEATURE_GPS)`, whereas repeated checks at run time can use `sensors(SENSOR_GPS)`.